python/shotgun_fields/image_widget.py

```diff
@@ -279,24 +318,53 @@ def _replace_image(self):
         """
         self._upload_image()
 
-    def _scale_pixmap(self):
+    def _scale_pixmap(self, pixmap):
         """
         Scale the pixmap in preparation for display.
         """
-        scaled_pixmap = self._pixmap.scaled(
+        return pixmap.scaled(
```

Comment:
Agreed. Here's a test: 

`KeepAspectRatio`
----
widget:
![screen shot 2016-07-19 at 8 11 34 am](https://cloud.githubusercontent.com/assets/2604646/16949383/76dd07c8-4d88-11e6-9501-8c44ab5ac2f2.png)
delegate (tall & wide):
![screen shot 2016-07-19 at 8 09 20 am](https://cloud.githubusercontent.com/assets/2604646/16949380/76d8d950-4d88-11e6-9dac-4275adedff2a.png)

`KeepAspectRatioByExpanding`
----
widget:
![screen shot 2016-07-19 at 8 11 13 am](https://cloud.githubusercontent.com/assets/2604646/16949382/76daf5be-4d88-11e6-8261-f1fe58463c33.png)
delegate (tall & wide):
![screen shot 2016-07-19 at 8 11 07 am](https://cloud.githubusercontent.com/assets/2604646/16949381/76d8f0b6-4d88-11e6-9119-aad9dd50dc0b.png)

Obviously the way it's currently implemented, using the stand alone display/edit widget looks better... though I'm sure it could be smarter. As for the delegates, I think wide images are better when displaying the full image. The tall image is equally useless in either case I think. :)

I'm inclined to keep it as-is unless you feel strongly it should be changed.